test/jdk/java/util/regex/RegExTest.java

```diff
@@ -74,146 +67,35 @@
 import java.util.stream.IntStream;
 import java.util.stream.Stream;
 
+import org.testng.annotations.Test;
+import org.testng.Assert;
+
+
 import jdk.test.lib.RandomFactory;
 
+//Asserts in JUnit5
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertNotEquals;
+import static org.testng.Assert.assertNotSame;
+import static org.testng.Assert.assertThrows;
+import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.fail;
+import static org.testng.Assert.expectThrows; //Replaced by assertThrows in JUnit5
+
```

Comment:
Slightly odd to mention JUnit 5 here, since this is being converted to a TestNG test. Are these notes for yourself for future work?

Reply:
Ah yes. I'll pull that.